stable/wordpress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: wordpress
-version: 5.11.0
```

Comment:
I see that it goes form 5.11 to 5.13. Why not 5.12?

Reply:
Because it created another PR bumping from `5.11` to `5.12`, so we need to take into account the order before merge

Reply:
This is right. I just opened 2 PR, so this one have one more version jump